Dockerfile

```diff
@@ -1,5 +1,4 @@
-# go-exploitdb v0.4.5
-FROM vuls/go-exploitdb@sha256:9f18d41d3f1a75d851a6126dba8aff873d1d2bb2ded78cbd6d0150cd0dba655d
+FROM vuls/go-exploitdb:v0.4.5
```

Comment:
please don't change this, we want to use image digests on purpose, to avoid re-tagging issues.

Reply:
I changed the amd64 digest for the manifest list of the multi platform image.

Reply:
For better dependabot support, you should use `vuls/go-exploitdb:v0.4.5@sha256:9f18d41d3f1a75d851a6126dba8aff873d1d2bb2ded78cbd6d0150cd0dba655d` format.

Reply:
Added the version to the base image for better dependabot support.